content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst

```diff
@@ -0,0 +1,233 @@
+      :alt: View the Product Barcodes page from inventory settings.
+
+.. _barcode/operations/lot-setup-on-product:
+
+After activating lots and serial number tracking in the settings page, specify that this feature is
+to be applied on each product by navigating to the :guilabel:`Inventory` tab on the product form.
+Under :guilabel:`Tracking`, choose the :guilabel:`By Lots` radio button.
+
+.. image:: gs1_usage/track-by-lots.png
+   :align: center
+   :alt: Enable product tracking by lots in the "Inventory" tab of the product form.
+
+Scan barcode on receipt
+-----------------------
+
+To ensure accurate lot interpretation in Odoo on product barcodes scanned during a receipt
+operation, navigate to the :menuselection:`Barcode` app to manage the :ref:`receipt picking process
+<barcode/operations/scan-received-products>`.
+
+From the :guilabel:`Barcode Scanning` dashboard, click the :guilabel:`Operations` button, then the
+:guilabel:`Receipts` button to view the list of vendor receptions to process. Receipts generated
+from :abbr:`POs (Purchase Orders)` are listed, but new receipt operations can also be created
+directly through the :menuselection:`Barcode` app using the :guilabel:`Create` button.
+
+On the list of receipts, click on the warehouse operation (`WH/IN`) and scan product barcodes and
+lot numbers with a barcode scanner. The scanned product then appears on the list. Use the
+:guilabel:`✏️ (pencil)` button to open a window and manually enter quantities for specific lot
+numbers.
+
+.. example::
+   After placing a :abbr:`PO (Purchase Order)` for fifty apples, navigate to the associated receipt
+   in the *Barcode* app.
+
+   Scan the barcode containing the |GTIN|, quantity, and lot number. For testing with a barcode
+   scanner, below is an example barcode for the fifty Fuji apples in Lot 2.
+
+   .. list-table::
+      :widths: 50 50
+      :header-rows: 1
+      :stub-columns: 1
+
+      * - 50 Fuji apples in Lot0002
+        -
+      * - 2D Matrix
+        - .. image:: gs1_usage/fuji-apples-barcode.png
+             :alt: 2D matrix of GS1 barcode of 50 fuji apples with an assigned lot number.
+      * - |AI| (product)
+        - 01
+      * - GS1 Barcode (product)
+        - 20611628936004
+      * - |AI| (quantity)
+        - 30
+      * - GS1 Barcode (quantity)
+        - 00000050
+      * - |AI| (lot)
+        - 10
+      * - GS1 Barcode (lot #)
+        - LOT0002
+      * - Full GS1 barcode
+        - 01206116289360043 000000050 10LOT0002
+
+   :ref:`If the configuration is correct <barcode/operations/troubleshooting>`, `50/50`
+   :guilabel:`Units` processed will be displayed and the :guilabel:`Validate` button turns green.
+   Click the :guilabel:`Validate` button to complete the reception.
+
+   .. image:: gs1_usage/receive-50-apples.png
+      :align: center
+      :alt: Scan the barcode for a product on the reception picking page in the *Barcode* app.
+
+.. _barcode/operations/quantity-ex:
+
+Product + non-unit quantity
```

Comment:
Same note for the similar heading above. But if you want to stick with this style heading then feel free, it's up to you